packages/workbox-background-sync/src/lib/QueueStore.ts

```diff
@@ -49,7 +49,8 @@ export class QueueStore {
   constructor(queueName: string) {
     this._queueName = queueName;
     this._db = new DBWrapper(DB_NAME, DB_VERSION, {
-      onupgradeneeded: this._upgradeDb,
+      // TODO (philipwalton): Should this be bound?
+      onupgradeneeded: this._upgradeDb, // eslint-disable-line @typescript-eslint/unbound-method
```

Comment:
@philipwalton

Reply:
Similar comment to above. It feels strange that we need to explicitly bind all these functions. Is this one of the rules enforced by Google TS style?

Reply:
I don't think we need to enforce this one. I can disable it globally in our repo if you'd like.

I just wanted to confirm before doing so.